packages/postgres/src/pool.test.ts

```diff
@@ -254,9 +253,13 @@ describe('@prairielearn/postgres', function () {
     });
   });
 
-  describe('queryCursor', () => {
+  describe('queryValidatedCursor', () => {
     it('returns zero rows', async () => {
-      const cursor = await queryCursor('SELECT * FROM workspaces WHERE id = 10000;', {});
+      const cursor = await queryValidatedCursor(
+        'SELECT * FROM workspaces WHERE id = 10000;',
+        {},
```

Comment:
Nit: for symmetry with `queryRow`/`queryRows`, consider making params argument optional.

Reply:
#12539